app/apptesting/concentrated_liquidity.go

```diff
@@ -41,6 +41,30 @@ func (s *KeeperTestHelper) PrepareConcentratedPoolWithCoinsAndFullRangePosition(
 	return clPool
 }
 
+// createConcentratedPoolsFromCoinsWithSwapFee creates CL pools from given sets of coins and respective swap fees.
+// Where element 1 of the input corresponds to the first pool created, element 2 to the second pool created etc.
+func (s *KeeperTestHelper) CreateConcentratedPoolsAndFullRangePositionWithSwapFee(poolDenoms [][]string, swapFee []sdk.Dec) {
+	for i, curPoolDenoms := range poolDenoms {
+		s.Require().Equal(2, len(curPoolDenoms))
+		clPool := s.PrepareCustomConcentratedPool(s.TestAccs[0], curPoolDenoms[0], curPoolDenoms[1], DefaultTickSpacing, swapFee[i])
+		fundCoins := sdk.NewCoins(sdk.NewCoin(curPoolDenoms[0], DefaultCoinAmount), sdk.NewCoin(curPoolDenoms[1], DefaultCoinAmount))
+		s.FundAcc(s.TestAccs[0], fundCoins)
+		s.CreateFullRangePosition(clPool, fundCoins)
+	}
+}
+
+// createConcentratedPoolsFromCoins creates CL pools from given sets of coins (with zero swap fees).
+// Where element 1 of the input corresponds to the first pool created, element 2 to the second pool created etc.
+func (s *KeeperTestHelper) CreateConcentratedPoolsAndFullRangePosition(poolDenoms [][]string) {
+	for _, curPoolDenoms := range poolDenoms {
+		s.Require().Equal(2, len(curPoolDenoms))
+		clPool := s.PrepareCustomConcentratedPool(s.TestAccs[0], curPoolDenoms[0], curPoolDenoms[1], DefaultTickSpacing, sdk.ZeroDec())
+		fundCoins := sdk.NewCoins(sdk.NewCoin(curPoolDenoms[0], DefaultCoinAmount), sdk.NewCoin(curPoolDenoms[1], DefaultCoinAmount))
+		s.FundAcc(s.TestAccs[0], fundCoins)
+		s.CreateFullRangePosition(clPool, fundCoins)
+	}
```

Comment:
cant we use CreateConcentratedPoolsAndFullRangePositionWithSwapFee here?

Reply:
fixed